client/Bootstrap/GraphQLClient/index.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+import { SubscriptionClient } from 'subscriptions-transport-ws';
+import { WebSocketLink } from '@apollo/link-ws';
+import config from '../../../config';
+
+const client = new SubscriptionClient(config.GRAPHQL_ENDPOINT, {
```

Comment:
Is the intent to have one Client used across the whole UI? As mentioned, we will have two GQL sources to query, the Strimzi-api, as well as the config for the UI. Given these two endpoints cannot be federated (as they are pluggable), if we want to have a single client, we could use multiple links with context to direct queries to one or the other.

I would also suggest not referencing a config value for now. I would suggest this is hardcoded to `/api` (with a second link for config, again hardcoded to `/config`). Thoughts?

Reply:
Given the BFF pattern, I think hardcoding the endpoints makes sense.

Multiple links makes sense, especially if we can use a directive to switch between them.

Reply:
Agreed with multiple links, the query context can be used to decide where the request should be directed to